tsconfig.json

```diff
@@ -14,7 +14,6 @@
   "exclude": [
     "node_modules",
     "dist",
-    "src/**/*.spec.ts",
```

Comment:
Intended change?

Reply:
Yeah, I'm not sure why it was exclude. Removing this line helped me ensure my tests were correct